### ISSUE TITLE
Feature request to print the layouts

### ISSUE THREAD

**Issue by svenski** [Ciantic/qmkmapper#3]
This is a great tool and has helped me iterate on changes for my new minidox, so thank you. 

It would be great if there was a way to print all the layouts to a document / system print. I'm currently print screening and cropping + copy pasting. :man_shrugging:

**Comment by Ciantic** [comment]
Interesting, do you use the VScode extension? Printing from that is probably out of scope but for me the web page version of this already works surprisingly well for printing:

http://ciantic.github.io/keymapceditor/#ergodox_ez|us|example

Shows me this in Chrome printing mode:

![image](https://user-images.githubusercontent.com/64731/74785852-02a24000-52b4-11ea-8bdb-9f1a4596213a.png)

And you can if you for some reason want to turn on the background too.

**Comment by svenski** [comment]
Thanks for the prompt reply. 

I did try the vs code extension, but you have to print out all the layers separately, and if you want them on a single cheat sheet, you'll have to somehow merge them. Granted, it's not a big job but it slows down iterations. I was hoping that there was some typescript foo to assemble the layouts on the same page. I had a quick look but my typescript is not up to scratch and wasn't sure how to even set up a dev env for it... 

I realised providing this example might be helpful: see the "Print Layout" link in the Ergodox EZ configurator: https://configure.ergodox-ez.com/ergodox-ez/layouts/default/latest/0

**Comment by Ciantic** [comment]
Yes, I will try to do that for the web page version. 

(I probably won't implement this for VSCode extension, because print preview in it is not probably easy to do.)

**Comment by svenski** [comment]
I also noticed that the drop down of keyboards (first dropdown) is "semi-sorted" alphabetically. I suspect whoever put it together started with an alphabetic but then other keys were added after that. It would be great if this was sorted -- I've scrolled up and down that list a couple of time now. Even better if I could just start typing at the top - sorry feature creep request!

**Comment by svenski** [comment]
Oh, maybe I should've entered a new issue for that, let me know -- happy to do that.

**Comment by Ciantic** [comment]
Yup, this is very rough around the edges. Feel free to add new issues here if you find, my projects languishes until I see a need to come back to them. With this I have to come back to it periodically because VS Code updates and what not.
